### PR TITLE
feat: add listbox search

### PR DESCRIPTION
**Pull request by stoffeastrom**
## Motivation

Add basic listbox search.

image

### CODE REVIEW

File: apis/nucleus/src/components/listbox/ListBoxRow.jsx [hunk 1]
```diff
@@ -34,6 +34,9 @@ const useStyles = makeStyles(theme => ({
   X: {
     background: theme.palette.background.darker,
   },
+  highlighted: {
+    backgroundColor: '#ffce26',
```
Comment (miralemd) [inline apis/nucleus/src/components/listbox/ListBoxRow.jsx]:
```suggestion
    backgroundColor: '#FFC72A',
```

**Comment by stoffeastrom** [comment]
Updated with highlight color

image

File: apis/nucleus/src/components/listbox/ListBoxSearch.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import { Grid, TextField } from '@nebula.js/ui/components';
+import Search from '@nebula.js/ui/icons/search';
+
+import { makeStyles } from '@nebula.js/ui/theme';
+
+import LocaleContext from '../../contexts/LocaleContext';
+
+const useStyles = makeStyles((/* theme */) => ({
+  gridContainer: {
+    padding: '0 8px 0 8px',
```
Comment (miralemd) [inline apis/nucleus/src/components/listbox/ListBoxSearch.jsx]:
```suggestion
   padding: theme.spacing(0, 1, 0, 1),
```

File: apis/nucleus/src/components/listbox/ListBoxSearch.jsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  },
+  gridItem: {
+    padding: '0 8px 0px 8px',
```
Comment (miralemd) [inline apis/nucleus/src/components/listbox/ListBoxSearch.jsx]:
```suggestion
    padding: theme.spacing(0, 1, 0, 1),
```

**Pull request merged**